scripts/create_backend_sandbox.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# expect 1 argument to the script
+if [ $# -ne 1 ]; then
+  echo 1>&2 "$0: expected 1 arguments, received $#"
+  exit 2
+fi
+
+ENVIRONMENT=$1
+COMPONENT="sandbox"
+AWS_ACCOUNT_ID="$(aws sts get-caller-identity --query Account --output text)"
+AWS_REGION="eu-west-2"
+PROJECT="nhs-notify"
+GROUP="nhs-notify-template-management-nonprod"
```

Comment:
would this still work if we gave it a group that didn't clash with an existing group in the internal repo? seems safer to do that if possible

Reply:
I'm not fully sure of everything `group` is for really - in this repo all I can see is it being used for tagging. In internal, it's used to pick the tfvars file. I think I added this when I was going down the route of trying to deploy this through the internal repo, and this would have been the right file to use. But I don't know if it's even needed here apart from for tagging.

Reply:
Maybe worth seeing what platform folk think when they review it